### PR TITLE
Issue/#226 buy sell UI

### PR DESCRIPTION
**Pull request by marwan-ayadi**
# Related Issue
- #226 

# Proposed changes
- add buy/sell ui elements in stock-detail page 

# Reviewer(s)
 - @Alessandro100

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="mat-typography" style="background-color: #f8f8fb">
```
Comment (Alessandro100) [inline client/src/index.html:18]: this is bad css practice, we should put this style property in styles.scss file

**Review by Alessandro100 — CHANGES_REQUESTED** [review]
It looks good, but there are a couple of things missing! 
image
Here if the total value is positive, the text should be green (red if its negative).

Secondly, in the story it also asks to make the UI if the user doesn't own any stocks. Check the issue and you'll see what I mean.

I also left a small css change

File: client/src/index.html (hunk 1 above)
Comment (marwan-ayadi) [reply client/src/index.html]: resolved

**Review by Alessandro100 — APPROVED** [review]
Nice changes

**Pull request merged**